source/_posts/potential-patent-infringement.md

```diff
@@ -0,0 +1,48 @@
+---
+title: How to check whether your product causes potential patent infringement
```

Comment:
Image should be optimized with Alt text

Reply:
We already have alt text added to image.